Modelica/Resources/C-Sources/ModelicaFFT.c

```diff
@@ -1,7 +1,7 @@
 /* ModelicaFFT.c - FFT functions
 
-   Copyright (C) 2015-2020, Modelica Association and contributors
-   Copyright (C) 2003-2010, Mark Borgerding
+   Copyright (C) 2015-2024, Modelica Association and contributors
+   Copyright (C) 2003-2024, Mark Borgerding
```

Comment:
```suggestion
   Copyright (C) 2003-2010, Mark Borgerding
```
As far as I know there hasn't been any action from Mark; so don't update that line.